audio-worklet/design-pattern/awn/audio-worklet-processor.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * A simple demonstration of WASM-powered AudioWorkletProcessor.
+ *
+ * @class WASMAudioWorkletProcessor
+ * @extends AudioWorkletProcessor
+ */
+class WASMAudioWorkletProcessor extends AudioWorkletProcessor {
+
+  constructor() {
+    super();
+
+    // Allocate the buffer for the heap access. Start with stereo, but it can
+    // be expanded up to 32 channels.
```

Comment:
I thought there was already a constant for the max number of channels.  Can it be used here?  If not, just add a comment that 32 is the min required number channels to be supported.

Reply:
Done.